homework/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="main.css">
+  <title>Javascript3</title>
+</head>
+
+<body>
+  <div>
+
+    <header>
+      <h1>HYF Repositories</h1>
+      <button>Javascript3</button>
+    </header>
+
+
+
+    <div class="main__input-container">
+      <input type="text" id="search" class="main__input" placeholder="Repositories">
+      <span>
+        <button class="searchButton">Search</button>
```

Comment:
Then you can remove this button too. You'll just load the repos when the page loads - so you don't need a trigger (event) for that. Next, to display the repo details, you'll just use the event from when the user changes the value of the select list.